DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -591,6 +594,145 @@ private void PostUIInit()
             GameProcessLogic.GameProcessExited += SharedUILogic_GameProcessExited;
         }
 
+        private void ConnectionManager_ChannelMessageOfTheDayComplete(object sender, EventArgs e)
+        {
+            RequestChannelList(null);
+            gameChanneListTimer = new Timer(RequestChannelList, null, 30000, 30000);
+        }
+
+        private void RequestChannelList(object state)
+        {
+            string pattern = gameCollection.GetGameListPatternFromIdentifier(localGameID);
+            connectionManager.RequestChannelList(pattern);
+        }
+
+        private void ConnectionManager_ChannelListReceived(object sender, ChannelTopicEventArgs e)
+        {
+            Logger.Log($"GameBroadcastChannel_ChannelListReceived ** {e.ChannelName}, {e.Topic}");
+
+            var cncnetManager = (CnCNetManager)sender;
+
+            // Ensure the topic starts with "GAME " as expected
+            if (!e.Topic.StartsWith("GAME "))
+                return;
+
+            // Extract the game information from the topic string
+            string msg = e.Topic.Substring(5); // Cut out "GAME " part
+            string[] splitMessage = msg.Split(new char[] { ';' });
```

Comment:
It would be great if you added a comment with an example valid topic for reference.